pkg/controller/jobframework/base_webhook.go

```diff
@@ -67,7 +67,7 @@ func (w *BaseWebhook) Default(ctx context.Context, obj runtime.Object) error {
 	log := ctrl.LoggerFrom(ctx)
 	log.V(5).Info("Applying defaults")
 	ApplyDefaultLocalQueue(job.Object(), w.Queues.DefaultLocalQueueExist)
-	if err := ApplyDefaultForSuspend(ctx, job, w.Client, w.ManageJobsWithoutQueueName, w.ManagedJobsNamespaceSelector); err != nil {
+	if err := ApplyDefaultForSuspend(ctx, w.Client, job, w.Client, w.ManageJobsWithoutQueueName, w.ManagedJobsNamespaceSelector); err != nil {
```

Comment:
We're already passing w.Client as a parameter.

Reply:
Wow, thanks!